src/main/java/org/folio/consortia/controller/ErrorHandling.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.consortia.controller;
+
+import org.folio.consortia.exception.ResourceNotFoundException;
+import org.folio.tenant.domain.dto.Errors;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+import static org.folio.consortia.utils.ErrorHelper.ErrorCode.NOT_FOUND_ERROR;
+import static org.folio.consortia.utils.ErrorHelper.ErrorCode.VALIDATION_ERROR;
+import static org.folio.consortia.utils.ErrorHelper.createInternalError;
+
+@RestControllerAdvice
+public class ErrorHandling {
```

Comment:
I propose to rename ExceptionHandlingController

Reply:
Ok